src/main/java/com/epam/grid/engine/entity/job/slurm/SlurmJob.java

```diff
@@ -0,0 +1,254 @@
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * This class represents SLURM job.
+ */
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class SlurmJob {
+    /**
+     * Account the job ran under.
+     */
+    private String account;
+    /**
+     * Trackable RESources per node.
+     */
+    private String tresPerNode;
+    /**
+     * Minimum number of CPUs (processors) per node requested by the job.
+     * This reports the value of the srun --mincpus option with a default value of zero.
+     */
+    private int minCpus;
+    /**
+     * Minimum size of temporary disk space (in MB) requested by the job.
+     */
+    private int minTmpDisk;
+    /**
+     * The time of job termination, actual or expected.
+     */
+    private LocalDateTime endTime;
+    /**
+     * Features required by the job.
+     */
+    private String features;
+    /**
+     * Group name of the job. (Valid for jobs only)
+     */
+    private String groupName;
+    /**
+     * Can the compute resources allocated to the job be over subscribed by other jobs.
+     * The resources to be over subscribed can be nodes, sockets, cores,
+     * or hyperthreads depending upon configuration. The value will be "YES" if the job
+     * was submitted with the oversubscribe option or the partition is configured with
+     * OverSubscribe=Force, "NO" if the job requires exclusive node access, "USER"
+     * if the allocated compute nodes are dedicated to a single user, "MCS"
+     * if the allocated compute nodes are dedicated to a single security class
+     * (See MCSPlugin and MCSParameters configuration parameters for more information),
+     * "OK" otherwise (typically allocated dedicated CPUs).
+     */
+    private String overSubscribe;
+    /**
+     * Job ID. This will have a unique value for each element of job arrays and each component of heterogeneous jobs.
+     */
+    private int jobId;
+    /**
+     * Job or job step name.
+     */
+    private String name;
+    /**
+     * Comment associated with the job.
+     */
+    private String comment;
+    /**
+     * Timelimit for the job or job step.
+     */
+    private String timelimit;
+    /**
+     * Minimum size of memory (in MB) requested by the job.
+     */
+    private String minMemory;
+    /**
+     * List of node names explicitly requested by the job.
+     */
+    private String reqNodes;
+    /**
+     * The command to be executed.
+     */
+    private String command;
+    /**
+     * Priority of the job (converted to a floating point number between 0.0 and 1.0). Also see prioritylong.
+     */
+    private double priority;
+    /**
+     * Quality of service associated with the job.
+     */
+    private String qos;
+    /**
+     * The reason a job is in its current state. See the JOB REASON CODES section below for more information.
```

Comment:
If some `JOB REASON SECTION` is mentioned here, it would be nice having a direct link to it, because  right now it is not clear at all